### PR TITLE
Cleaning up and updating content in the project repo

### PR DESCRIPTION
**Pull request by mheadd**
This pull request cleans up some general items in the project repo to reflect current state, to update outdated references and to remove unneeded items.

### CODE REVIEW

**Comment by mheadd** [comment]
@sztaylorakgov Wanted to ask for your review on the project risk section - the criticality and status of several seemed off and I wanted to update them to reflect where we are now.

**Comment by mheadd** [comment]
@sztaylorakgov I think it might make sense to look at all of the risks to make sure we think that we have them rated appropriately.

File: ACRONYMS.md [hunk 1]
```diff
@@ -1,17 +1,376 @@
...
+| CI                          | DKC CHIP /Title 21 funding  (Subtypes S2, H2, and CP)-(Incarceration Medicaid Subtype)                                                                                                                                                           | DPA     |
+| CITC                        | Cook Inlet Tribal Council                                                                                                                                                                                                                        | SOA     |
+| CLNO                        | Client Notes (EIS screen)                                                                                                                                                                                                                        | DPA     |
+| CLNO                        | Client Notes (EIS screen)                                                                                                                                                                                                                        | DPA     |
```
Comment (edmullen) [inline ACRONYMS.md]: This CLNO is a duplicate.

File: README.md [hunk 2]
```diff
@@ -88,8 +88,8 @@ We will know we are successful, if we can increase worker productivity and moral
...
 | There is a risk that vendors won't be able to easily work with the legacy ARIES code and existing authentication methods. | 5  | Not started  |
 | There is a landscape of existing solutions that could potentially be leveraged instead of building from scratch.  |  2 |  Not started |
-| There is a risk that the ATO process will prevent us from being able to deploy continuously to a production environment.  | 2  | Derisked  |
-| There is a risk that by having multiple vendors contributing to the same codebase, we increase the complexity of integration. There could be problems that arise specifically from the logistics and timing of multiple contributors.   | 2  |  Derisked |
+| There is a risk that the ATO process will prevent us from being able to deploy continuously to a production environment.  | 5  | In progress  |
+| There is a risk that by having multiple vendors contributing to the same codebase, we increase the complexity of integration. There could be problems that arise specifically from the logistics and timing of multiple contributors.   | 3  |  Not started |
```
Comment (edmullen) [inline README.md]: I'm not sure if I'd say this is "not started". I agree that it should not be "Derisked". But some of the VSTS prep and several other technical and design choices are working towards this. But, I'm not firm on this point. I could see it either way.

Comment (mheadd) [reply README.md]: Yeah, I guess my initial thinking on this was that it wouldn't be derisked until there was a 2nd vendor contributing code through VSTS, but you're right that this work has been started. I think changing from `Not started` to `In progress`makes the most sense.

Comment (mheadd) [reply README.md]: @edmullen Fixed it.

**Pull request merged**